packages/gestalt/src/Button.js

```diff
@@ -62,7 +61,8 @@ export default function Button(props: Props): Element<'button'> {
   } = props;
   const { name: colorSchemeName } = useColorScheme();
   // We need to make a few exceptions for accessibility reasons in darkMode for red buttons
-  const isDarkModeRed = colorSchemeName === 'darkMode' && color === 'red';
+  const DARK_MODE = colorSchemeName === 'darkMode';
```

Comment:
Caps/underscore doesn't go with the naming style of the rest of the codebase. Could you name this like `isDarkMode` or `inDarkMode` maybe?